.eslintrc.json

```diff
@@ -1,4 +1,14 @@
-// TODO: Since we already have eslintrc.js we should remove this or move this part to eslintrc.js
 {
-  "extends": "next/core-web-vitals"
+  "root": true,
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "next/core-web-vitals",
+    "prettier"
+  ],
+  "plugins": ["@typescript-eslint"],
+  "rules": {
+    "no-console": "error",
+    "no-shadow": "error"
```

Comment:
Using the same variable name in different scopes isn't a problem at self. The problem is that developer can forget about it and delete variable inside inner scope or not setup it, and ESLint will not help in that case, but there will be a bug.

Reply:
It's not "Outdated". File just was renamed